PProPanel/jsx/Premiere.jsx

```diff
@@ -392,88 +410,97 @@ $._ext_PPRO={
 			    app.encoder.startBatch();
 			}
 
-			var proj_path   	= new File(app.project.path);
-			var out_path    	= new File("~/Desktop");
-			var sep         	= '\\';
-			var out_preset_path	= "C:\Program Files\Adobe\Adobe Media Encoder CC 2014\MediaIO\systempresets\58444341_4d584658\XDCAMHD 50 NTSC 60i.epr";
+			var projPath   	= new File(app.project.path);
+			var outputPath  = Folder.selectDialog("Choose the output directory");
+
+			if (outputPath != null && projPath.exists){
 			
-			if (qe.platform == 'Macintosh') {
-				var out_preset_path = "/Applications/Adobe Premiere Pro CC 2014/Adobe Premiere Pro CC 2014.app/MediaIO/systempresets/58444341_4d584658/XDCAMHD 50 NTSC 60i.epr";
-				sep = '/';
+				if (Folder.fs == 'Macintosh') {
+					var outputPresetPath = "/Applications/Adobe\ Premiere\ Pro\ CC\ 2015/Adobe\ Premiere\ Pro\ CC\ 2015.app/Contents/MediaIO/systempresets/58444341_4d584658/XDCAMHD\ 50\ NTSC\ 60i.epr";
```

Comment:
I don't think I've given away too much. If any rumor sites pop up, stating that they know the name and number of the next release of PPro—it's all on me. :)